### ISSUE TITLE
KeyError: 'product_uom'

### ISSUE THREAD

**Issue by max3903** [pledra/odoo-product-configurator#71]
Impacted versions:
9.0 and 10.0

Steps to reproduce:
* Install the sale_margin module
* Configure a product through the configuration wizard on a sales order
* Click Next on the last step

Current behavior:
```
Traceback (most recent call last):
  File "/home/mchambreuil/odoo/10.0/env/lib/python2.7/site-packages/odoo/http.py", line 638, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/home/mchambreuil/odoo/10.0/env/lib/python2.7/site-packages/odoo/http.py", line 675, in dispatch
    result = self._call_function(**self.params)
  File "/home/mchambreuil/odoo/10.0/env/lib/python2.7/site-packages/odoo/http.py", line 331, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/home/mchambreuil/odoo/10.0/env/lib/python2.7/site-packages/odoo/service/model.py", line 119, in wrapper
    return f(dbname, *args, **kwargs)
  File "/home/mchambreuil/odoo/10.0/env/lib/python2.7/site-packages/odoo/http.py", line 324, in checked_call
    result = self.endpoint(*a, **kw)
  File "/home/mchambreuil/odoo/10.0/env/lib/python2.7/site-packages/odoo/http.py", line 933, in __call__
    return self.method(*args, **kw)
  File "/home/mchambreuil/odoo/10.0/env/lib/python2.7/site-packages/odoo/http.py", line 504, in response_wrap
    response = f(*args, **kw)
  File "/home/mchambreuil/odoo/10.0/env/lib/python2.7/site-packages/odoo/addons/web/controllers/main.py", line 882, in call_button
    action = self._call_kw(model, method, args, {})
  File "/home/mchambreuil/odoo/10.0/env/lib/python2.7/site-packages/odoo/addons/web/controllers/main.py", line 870, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/home/mchambreuil/odoo/10.0/env/lib/python2.7/site-packages/odoo/api.py", line 681, in call_kw
    return call_kw_multi(method, model, args, kwargs)
  File "/home/mchambreuil/odoo/10.0/env/lib/python2.7/site-packages/odoo/api.py", line 672, in call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/home/mchambreuil/odoo/10.0/src/odoo-product-configurator/product_configurator_wizard/wizard/product_configurator.py", line 728, in action_next_step
    return self.action_config_done()
  File "/home/mchambreuil/odoo/10.0/src/odoo-product-configurator/product_configurator_wizard/wizard/product_configurator.py", line 804, in action_config_done
    'name': variant.display_name
  File "/home/mchambreuil/odoo/10.0/env/lib/python2.7/site-packages/odoo/addons/mail/models/mail_thread.py", line 274, in write
    result = super(MailThread, self).write(values)
  File "/home/mchambreuil/odoo/10.0/env/lib/python2.7/site-packages/odoo/models.py", line 3550, in write
    self._write(old_vals)
  File "/home/mchambreuil/odoo/10.0/env/lib/python2.7/site-packages/odoo/models.py", line 3668, in _write
    field.write(self.with_context(rel_context), vals[name])
  File "/home/mchambreuil/odoo/10.0/env/lib/python2.7/site-packages/odoo/fields.py", line 2172, in write
    comodel.create(act[2])
  File "/home/mchambreuil/odoo/10.0/env/lib/python2.7/site-packages/odoo/addons/sale_margin/models/sale_order.py", line 50, in create
    product_uom_id = self.env['product.uom'].browse(vals['product_uom'])
KeyError: 'product_uom'
```

Expected behavior:
* The configured variant is added to the sales order

**Comment by max3903** [comment]
I upgraded my code and it works on 10.0, thanks to:
https://github.com/pledra/odoo-product-configurator/commit/54f0faee1edcb5e3e91a77f78fc7741918fa5d9b

**Comment by busyfritz** [comment]
Need 54f0fae fix backported to 9.0

**Comment by PCatinean** [comment]
@busyfritz it is backported already: https://github.com/pledra/odoo-product-configurator/commit/d4f9f31a5e1c7989b8ad1d54e88e9c3b62e32bf9
